pr-and-deploy-dependants/entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+if [ -z "$INPUT_DEPENDANTS" ]
+then
+  echo "Dependant repos must be defined"
+  return -1
+fi
+
+if [ $INPUT_DESTINATION_HEAD_BRANCH == "main" ] || [ $INPUT_DESTINATION_HEAD_BRANCH == "master" ]
+then
+  echo "Destination head branch cannot be 'main' nor 'master'"
+  return -1
+fi
+
+CLONE_DIR=$(mktemp -d)
+
+echo "Setting git variables"
+export API_TOKEN_GITHUB=${API_TOKEN_GITHUB//$'\n'}
+export GITHUB_TOKEN=$API_TOKEN_GITHUB
+export GOPRIVATE="github.com/voiapp"
+git config --global user.email "$INPUT_USER_EMAIL"
+git config --global user.name "$INPUT_USER_NAME"
+
+#wget -q  https://api.github.com/repos/cli/cli/releases/latest \
+#    && wget -q $(cat latest | grep linux_amd64.tar.gz | grep browser_download_url | grep -v .asc | cut -d '"' -f 4) \
+#    && tar -xvzf gh*.tar.gz \
+#    && mv gh*/bin/gh /usr/local/bin/ \
+#    && rm -fr *
+
+git config --global url."https://$BENDER_PAT@github.com/".insteadOf "https://github.com/"
+git config --global http.extraheader "PRIVATE-TOKEN: $BENDER_PAT"
+
+export IFS=","
+INDEX=1
+
+for repo in $INPUT_DEPENDANTS; do
+  BRANCH="$(cut -d',' -f$INDEX <<<"$INPUT_DESTINATION_BASE_BRANCH")"
+  echo "Cloning $repo"
+  git clone "https://$API_TOKEN_GITHUB@github.com/$repo.git" $repo
```

Comment:
Question: I'm confused by this line. Aren't we already using the `BENDER_PAT` (from l33?).
```suggestion
  git clone "https://github.com/$repo.git" $repo
```

Wouldn't that be enough?

Reply:
yep, fixed